Modern/wrapperPlugins/src/main/java/org/bonej/wrapperPlugins/AnisotropyWrapper.java

```diff
@@ -74,49 +75,51 @@
 	ContextCommand
 {
 
-	private static BinaryFunctionOp<RandomAccessibleInterval<BitType>, AxisAngle4d, List<Vector3d>> milOp;
+	private static BinaryFunctionOp<RandomAccessibleInterval<BitType>, AxisAngle4d, Vector3d> milOp;
 	private static UnaryFunctionOp<Matrix4d, Ellipsoid> quadricToEllipsoidOp;
 	private static UnaryFunctionOp<List<Vector3d>, Matrix4d> solveQuadricOp;
-	private static int ROTATIONS = 1_000;
-	private static int DEFAULT_LINES = 50;
+	/**
+	 * Default directions is 2_000 since that's roughly the number of points in
+	 * Poisson distributed sampling that'd give points about 5 degrees apart).
+	 */
+	private static int DEFAULT_DIRECTIONS = 2_000;
+	// Magic number, there are no good guesses what might be the right number of
+	// lines. Selected by fiddling around with the tool
+	private static int DEFAULT_LINES = 100;
```

Comment:
maybe formulate this more positively: "Number found to be sensible from experience with data at hand, may differ for other data" or similar?